### ISSUE TITLE
Decide on a date for Arduino & Raspberry Pi day.

### ISSUE THREAD

**Issue by ScarletHg** [ChelmsfordMakerspace/space#29]
Mostly dependent on #28. At the meeting we worked out that the 16th to the 22nd worked within the summer holidays while leaving enough time to fully prepare and run some marketing.

**Comment by ScarletHg** [comment]
Oops, this is a duplicate of an issue over in ChelmsfordMakerspace/ArduinoRPiDay2014.
